### DISCUSSION TITLE
cloudflare这初生，免费域名不让用API

### DISCUSSION THREAD

**Discussion by daiaji** [luolongfei/freenom#38] [General]
艹，DDNS就寄了。

**Comment by mapxn** [comment]
你可以DDNS到一个付费（子）域名上，然后免费域名CNAME记录 解析到前者。

**Reply by daiaji** [reply]
我买了xyz域名，本来用freenom的域名就是想白嫖，现在都付费了，那freenom的免费域名就没有必要用了。

**Reply by mapxn** [reply]
域名也可以用eu.org的，这个免费且支持DDNS。再CNAME到freenom的域名

**Reply by daiaji** [reply]
已经买了，不用多少有点可惜，到期再折腾好了。
